packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -215,4 +215,43 @@ void main() {
     expect(iconColor(expandedIconKey), _unselectedWidgetColor);
     expect(iconColor(collapsedIconKey), _accentColor);
   });
+
+  testWidgets('ExpansionTile maintainsState', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        theme: ThemeData(
+          platform: TargetPlatform.iOS,
+          dividerColor: _dividerColor,
+        ),
+        home: Material(
+          child: SingleChildScrollView(
+            child: Column(
+              children: const <Widget>[
+                ExpansionTile(
+                  title: Text('Tile 1'),
+                  initiallyExpanded: false,
+                  maintainsState: true,
+                  children: <Widget>[
+                    Text('Maintaining State'),
+                  ],
+                ),
+                ExpansionTile(
+                  title: Text('Title 2'),
+                  initiallyExpanded: false,
+                  maintainsState: false,
+                  children: <Widget>[
+                    Text('Discarding State'),
+                  ],
+                ),
+              ],
+            ),
+          ),
+        ),
+    ));
+
+    // This text should be there while ExpansionTile collapsed
+    expect(find.text('Maintaining State'), findsOneWidget);
```

Comment:
The tester shouldn't find this if it is not visible unless `skipOffstage: true` is specified.